SpliceAI.pm

```diff
@@ -268,11 +271,29 @@ sub run {
         $hash{'SpliceAI_cutoff'} = $result_flag;
       }
 
-      return ($self->{config}->{output_format} eq "json" || $self->{config}->{rest}) ?  {SpliceAI => \%hash} : \%hash;
+      $hash_aux{$data_value->{gene}} = \%hash;
+    }
+  }
+
+  return {} unless(%hash_aux);
+
+  my $result = {};
+
+  my $n_genes = scalar keys %hash_aux;
+  if($n_genes == 1) {
+    # Get the first and only gene from the hash of results
+    my $key_gene = (keys %hash_aux)[0];
+    $result = ($self->{config}->{output_format} eq "json" || $self->{config}->{rest}) ?  {SpliceAI => $hash_aux{$key_gene}} : $hash_aux{$key_gene};
+  }
+  elsif($n_genes > 1) {
+    # Compare genes from SpliceAI with the variant gene
+    my $genename = $tva->transcript->get_Gene->external_name;
```

Comment:
This requires a database connection and will cause problems when run with --offline. The G2P plugin also needs the gene symbol and gets it like this my $gene_symbol = $transcript->{_gene_symbol} || $transcript->{_gene_hgnc}; (https://github.com/Ensembl/VEP_plugins/blob/release/103/G2P.pm#L523)